content/blog/2023-11-13-announcing-localstack-30-general-availability/index.md

```diff
@@ -0,0 +1,186 @@
+---
```

Comment:
nit (if not already done): I guess we could make this blog pinned, and then unpin the v2.0 announcement blog. 👍